query.css

```diff
@@ -0,0 +1,356 @@
+    margin-left: 200px;
+    width: 60%;
+    height: auto;
+    letter-spacing: 0.37px;
+  }
+
+  .connect_button {
+    margin-left: 200px;
+    font-size: 20px;
+  }
+
+  .social ol {
+    margin-left: 200px;
+  }
+
+  .desktop-card,
+  .card {
+    background-color: white;
+    display: flex;
+    height: 400px;
+    width: 85%;
+    padding: 24px;
+
+    /* margin-bottom: 70px; */
+    border-radius: 24px;
+  }
+
+  .desktop-preview,
+  .preview {
+    grid-column: 1/-1;
+    justify-self: center;
+    align-self: center;
+  }
+
+  .desktop-preview img,
+  .preview img {
+    margin-top: 10px;
+    width: 100%;
+    height: 350px;
+  }
+
+  .desktop-details,
+  .details {
+    grid-column: 1/-1;
+    align-self: center;
+    margin-left: 50px;
+    padding: 10px;
+  }
+
+  .desktop-title,
+  .title {
+    margin: 10px 0;
+    color: #172b4d;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+  }
+
+  .desktop-info,
+  .info {
+    display: flex;
+    overflow: hidden;
+    color: #7a869a;
+  }
+
+  .desktop-info_item,
+  .info_item {
+    margin-right: 15px;
+    list-style-position: inside;
+  }
+
+  .desktop-info_item:first-child,
+  .info_item:first-child {
+    font-weight: bold;
+    list-style: none;
+    color: #344563;
+  }
+
+  .desktop-info_item:last-child,
+  .info_item:last-child {
+    margin-right: 0;
+  }
+
+  .desktop-details_content,
+  .details_content {
+    width: 80%;
+    margin: 10px 0;
+    padding: 10px 0;
+    color: #344563;
+  }
+
+  .desktop-catagories,
+  .catagories {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 20px 0;
+  }
+
+  .catagories .special {
+    display: flex;
+  }
+
+  .desktop-catagories ul li,
+  .catagories ul li {
+    padding: 8px;
+    color: #6070ff;
+    border-radius: 8px;
+    margin-right: 8px;
+    background: #ebebff;
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .desktop-default_button,
+  .default_button {
+    cursor: pointer;
+    font-size: 18px;
+    margin-top: 20px;
+    color: #396df2;
+    padding: 10px 20px;
+    background: #fff;
+    border-radius: 10px;
+    border: 1px solid #396df2;
+  }
+
+  .desktop-default_button:hover,
+  .default_button:hover {
+    color: #fff;
+    background: #396df2;
+  }
+
+  #order-4 {
+    order: 1;
+    flex-direction: row-reverse;
+  }
+
+  #order-2 {
+    order: 3;
+    flex-direction: row-reverse;
+  }
+
+  #order-1 {
+    order: 2;
+  }
+
+  #order-3 {
+    display: none;
+  }
+
+  /* end card */
+
+  /* about */
+  .about {
+    display: flex;
+    align-items: baseline;
+    width: 100%;
+  }
+
+  .about-left-section {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 40%;
+    height: 392px;
+    margin: 50px 0 0 80px;
+  }
+
+  .about-left-section .connect_button,
+  .about-left-section .social ol {
+    margin: 0;
+  }
+
+  .about-right-section {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-left: 200px;
+  }
+
+  .about_ul {
```

Comment:
:red_circle: Kindly notice that each language item has an icon at the left. Your design has no icon for this section as you can in this screenshot

Your Design
![yrnoicon](https://user-images.githubusercontent.com/26736582/200089031-3c0c4e9b-692d-4b6c-8b20-a7d09e973a6a.png)

Figma Design
![noarrows](https://user-images.githubusercontent.com/26736582/200089045-be715ea0-9ae9-4baa-a19e-b3bb70608aea.png)

:red_circle: Please make sure you include the icons as shown on this Figma design :arrow_up_small:

Reply:
:red_circle: Kindly notice that the bottom line is only applied to the last two items (Frameworks & Skills). Please make sure you remove the bottom line on the first item (language)

Your Design
![yrborderless](https://user-images.githubusercontent.com/26736582/200089456-013f0d6f-de18-459e-91b1-6c2c0cc43710.png)

Figma Design
![figborderless](https://user-images.githubusercontent.com/26736582/200089470-243f69ba-1d3c-4143-88d9-2489449fd4f5.png)

Reply:
- [ ] Please implement icons in the above section, asked to work by the previous reviewer :+1: